app/scripts/metamask-controller.js

```diff
@@ -1183,12 +1170,11 @@ module.exports = class MetamaskController extends EventEmitter {
    */
   setCurrentCurrency (currencyCode, cb) {
```

Comment:
I would almost rather move this whole function into the `currencyRateController`. We can do that later, but it shows a nice place where we can continue tightening up the Mm-controller, reduce it to the API-passing method `getAPI()`, and whatever equivalent we start to expose for the "untrusted API" (including the provider, accounts, etc).

Reply:
100%. This type of near-pass-through will definitely get nixed when we port MetaMaskController.